shell/browser/native_window_views.cc

```diff
@@ -212,7 +213,8 @@ NativeWindowViews::NativeWindowViews(const mate::Dictionary& options,
   window_state_watcher_.reset(new WindowStateWatcher(this));
 
   // Set _GTK_THEME_VARIANT to dark if we have "dark-theme" option set.
-  bool use_dark_theme = false;
+  bool use_dark_theme =
+      ui::NativeTheme::GetInstanceForNativeUi()->ShouldUseDarkColors();
   if (options.Get(options::kDarkTheme, &use_dark_theme) && use_dark_theme) {
```

Comment:
Not new to this PR, but now that I read it again -- are you sure we want the `&& use_dark_theme` clause here? Do we want to call `SetGTKDarkThemeEnabled()` here even if `use_dark_theme` is false?

Reply:
Good call, we should call it no matter what 👍